cypress_test/integration_tests/desktop/writer/scrolling_spec.js

```diff
@@ -57,7 +57,9 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Scroll through document',
 		desktopHelper.assertScrollbarPosition('horizontal', 430, 653);
 	});
 
-	it('Check if we jump the view on change of formatting mark', function() {
+	// TODO: fix this test, it seems like scrollWriterDocumentToTop is not scrolling to the top
+	// visually
```

Comment:
It seems to me that this is catching a real problem, I've filed https://github.com/CollaboraOnline/online/issues/11347 to track this & intend to look at it myself. If you test manually, exactly view jump happens, no problem with the test here. :-)

Reply:
Thanks :)